src/Draco.Compiler/Internal/Symbols/Metadata/MetadataPropertyAccessorSymbol.cs

```diff
@@ -0,0 +1,20 @@
+using System.Collections.Immutable;
+using System.Reflection.Metadata;
+using Draco.Compiler.Internal.Symbols.Generic;
+
+namespace Draco.Compiler.Internal.Symbols.Metadata;
+
+internal sealed class MetadataPropertyAccessorSymbol : MetadataMethodSymbol, IPropertyAccessorSymbol
```

Comment:
Document please

Reply:
I suppose then that this is either a getter, setter, or any other accessor function?